### PR TITLE
[AIDAPP-330]: Update the copyright text for the footer in Aiding App

### PR DESCRIPTION
**Pull request by ketan-canyon**
### Ticket(s) or GitHub Issue

- https://canyongbs.atlassian.net/browse/AIDAPP-330

### Technical Description

Update the copyright text for the footer in Advising App

### Any deployment steps required?

No

### Are any Feature Flags and/or Data Migrations that can eventually be removed Added?

No
_______________________________________________

### CODE REVIEW

File: portals/knowledge-management/src/Components/Footer.vue [hunk 1]
```diff
@@ -50,10 +50,11 @@
 
         <div class="flex w-full justify-center pb-4 pt-2">
             <span class="w-11/12 text-center text-xs lg:w-3/4 xl:w-7/12">
-                © 2016-{{ new Date().getFullYear() }} Canyon GBS LLC. All Rights Reserved. Canyon GBS™ and Aiding
-                App™ are trademarks of Canyon GBS LLC. For more information or inquiries, please visit our website at
+                © 2016-{{ new Date().getFullYear() }} Canyon GBS LLC. All Rights Reserved. Canyon GBS&reg; and Aiding
+                App&reg; are trademarks of Canyon GBS LLC. For more information or inquiries, please visit our website
+                at
```
Comment (Orrison) [inline portals/knowledge-management/src/Components/Footer.vue:55]: Though `&reg;` is typically fine in almost all cases, lets use the numeric version `&#174;` as it has the most compatibility between the two.

File: resources/views/filament/footer.blade.php [hunk 2]
```diff
@@ -48,12 +48,13 @@ class="block h-7 dark:hidden"
 
     <div class="flex w-full justify-center pb-4 pt-2">
         <span class="w-11/12 text-center text-xs lg:w-3/4 xl:w-7/12">
-            © 2016-{{ date('Y') }} Canyon GBS LLC. All Rights Reserved. Canyon GBS™ and Aiding App™ are trademarks
+            © 2016-{{ date('Y') }} Canyon GBS LLC. All Rights Reserved. Canyon GBS&reg; and Aiding App&reg; are
+            trademarks
```
Comment (Orrison) [inline resources/views/filament/footer.blade.php:52]: Though `&reg;` is typically fine in almost all cases, lets use the numeric version `&#174;` as it has the most compatibility between the two.

**Review by Orrison — CHANGES_REQUESTED** [review]

File: portals/knowledge-management/src/Components/Footer.vue (hunk 1 above)
Comment (ketan-canyon) [reply portals/knowledge-management/src/Components/Footer.vue]: Done

File: resources/views/filament/footer.blade.php (hunk 2 above)
Comment (ketan-canyon) [reply resources/views/filament/footer.blade.php]: Done

**Review by Orrison — APPROVED** [review]

**Pull request merged**